README.md

```diff
@@ -145,6 +146,45 @@ The optional `revision` value is printed on the title page and in the header, if
 
 The manufacturer name, which must be specified in `system.yml` data document, is also show on the title page.
 
+## Audit Checklists
+
+We include several checklists for various standards. 
+These are used by the `rdm gap` command to check output documents for appropriate references to a given standard. 
+
+For example ISO62304 requires that the device be classified according to the hazard levels 
+it could present.
+This requirement occurs in section 4.3.a of the standard.
+All of the 62304 checklists will include the following:
+```
+62304:4.3.a Software safety classification: system class
+```
+The gap analysis will report whether the key word `62304:4.3.a` appears in the documents. 
+Hopefully it does appear and it appears at the location where the hazard classification is
+described and justified. The tool only reports whether it is present or missing.
+The additional text `Software safety classification: system class` is not used by the tool.
+It is a pneumonic to help you locate or remember where in the standard you should be looking.
+
+You can create your own new or modified checklists.
+The checklists are read line by line using a very simple format:
+
+1. Leading white space is ignored.
+2. If the first non-white space is a '#' then the line is a comment and ignored.
+3. If the first non-white space is the word 'include' then the following text is
+treated as a file reference relative to the current checklist location.
+4. Otherwise the first non-white space word is treated as an expected key word.
+5. The keyword includes all text up to either a space character or the end of the line.
```

Comment:
Maybe this should be item 1, since it is the most important?

Reply:
Not sure which of the lines is "this".

Reply:
I was referring to the line directly above the comment. Thus, I think bullets 5 and 6 should be moved into the first two positions of the list. I think this will make it easier to understand the grammar if you haven't seen it before.